nostalgia/unzip.py

```diff
@@ -1,10 +0,0 @@
-import tqdm
-import sys
-import just
-import zipfile
-
-if __name__ == "__main__":
```

Comment:
I think it would be good practice to explain why this file was deleted (though I see the commit message says it's unnecessary), and maybe what it was once used for.

Reply:
It is unused simply. In favor of this I believe: nostalgia/nostalgia/extracter.py It's a generic way to unzip files into the right folder (using `ingest_settings`, see e.g. https://github.com/nostalgia-dev/nostalgia/tree/master/nostalgia/sources/google/__init__.py)

Reply:
Thanks -- I'm glad I asked.